pandas/compat/__init__.py

```diff
@@ -37,16 +34,26 @@
 PY37 = sys.version_info >= (3, 7)
 PYPY = platform.python_implementation() == 'PyPy'
 
-try:
-    import __builtin__ as builtins
-    import httplib
-except ImportError:
-    import builtins
-    import http.client as httplib
-
 from pandas.compat.chainmap import DeepChainMap
 
 
+# list-producing versions of the major Python iterating functions
+def lrange(*args, **kwargs):
```

Comment:
we want to completely remove these at some point as well

Reply:
Right. Just moving for now since their PY2 versions used `builtin`